app/assets/javascripts/discourse/app/models/user-badge.js

```diff
@@ -101,6 +98,8 @@ UserBadge.reopenClass({
       }
       if (json.meta) {
         userBadges.meta = json.meta;
+      } else {
+        userBadges.meta = {};
       }
```

Comment:
Should work as a one-liner?

```suggestion
        userBadges.meta = json.meta || {};
```

Reply:
SGTM

Reply:
I also added a default meta, since `max_favorites` is required for the templates.